openedx_webhooks/github/dispatcher/actions/utils.py

```diff
@@ -15,3 +23,63 @@ def find_issues_for_pull_request(jira, pull_request_url):
     """
     jql = 'project=OSPR AND cf[10904]="{}"'.format(pull_request_url)
     return jira.search_issues(jql)
+
+
+def _get_latest_commit_for_pull_request(repo_name_full: str, number: int) -> str:
+    data = _get_latest_commit_for_pull_request_data(repo_name_full, number)
+    commit = {}
+    if data:
+        commit = data[-1]
+    sha = commit.get('sha')
+    logger.debug(f"CLA: SHA %s", sha, repo_name_full)
+    return sha
+
+
+def _get_latest_commit_for_pull_request_data(repo_name_full: str, number: int) -> str:
+    """
+    Lookup the HEAD commit SHA for a pull request
+    """
+    url = f"https://api.github.com/repos/{repo_name_full}/pulls/{number}/commits"
```

Comment:
It looks like most of these types of functions (hitting the Github API) are actually located right inside `pr_tracking.py`.
Should I just move these there too? This had seemed like a reasonable place to stick them :shrug:

Reply:
It's fine to have them here.  the repo is kind of ad-hoc at this point.